core/src/test/scala/unit/kafka/server/GroupCoordinatorBaseRequestTest.scala

```diff
@@ -73,6 +74,17 @@ class GroupCoordinatorBaseRequestTest(cluster: ClusterInstance) {
     )
   }
 
+  protected def alterConsumerGroupOffsets(
+    groupId: String,
+    offsets: Map[TopicPartition, OffsetAndMetadata]
+  ): Unit = {
+    val resultFuture = cluster.createAdminClient().alterConsumerGroupOffsets(groupId, offsets.asJava).all
+
+    TestUtils.waitUntilTrue(() => {
+      resultFuture.isDone && !resultFuture.isCancelled && !resultFuture.isCompletedExceptionally
+    }, msg = s"Could not commit the offsets successfully. Last response ${resultFuture.get()}.")
```

Comment:
This is a bit weird. Is there a reason not to use `resultFuture.get()` with a timeout?

Reply:
Oh yes, you are right. Let me change this

Reply:
I was wondering how to get the error message from the future 🤔

Reply:
Hum... `.get` will throw the exception if the future is completed exceptionally or timed out. It should be enough, no?

Reply:
Ok let me remove the wait until true..